Fix ffi test - only run it if libc.6.so was loaded [skip ci]

TysonAndre opened the pull request:
This fixes the appveyor build, at least.

Not sure how a more permanent fix would be implemented. Feel free to amend this. This test passes locally on Linux

Fixes a bug in test added in
5661feb1ef929111ef7765672fcb49813eee86fc

https://ci.appveyor.com/project/php/php-src/builds/22456937/job/dyxp0ivgavs94w7n

```
========DIFF========
001+ Fatal error: Uncaught FFI\Exception: Failed loading 'libc.so.6' in C:\projects\php-src\ext\ffi\tests\bug77632.php:2
001- test
002+ Stack trace:
003+ #0 C:\projects\php-src\ext\ffi\tests\bug77632.php(2): FFI::cdef('int printf(cons...', 'libc.so.6')
004+ #1 {main}
005+   thrown in C:\projects\php-src\ext\ffi\tests\bug77632.php on line 2
========DONE========
FAIL Bug #77632 (FFI Segfaults When Called With Variadics) [C:\projects\php-src\ext\ffi\tests\bug77632.phpt] 
SKIP Bug #68996 (Invalid free of CG(interned_empty_string)) [C:\projects\php-src\ext\fileinfo\tests\bug68996.phpt] reason: Need Zend MM disabled
```

TysonAndre commented:
https://github.com/winlibs/glib/search?q=printf&unscoped_q=printf

My best guess is that it'd be found in php-install-dir/glib-2.dll on Windows? I'm not sure of that, though

nikic commented:
You might want to use zend_printf, see https://github.com/php/php-src/blob/e089d506d5c7716c62cee5232d32ab22d0ddde26/ext/ffi/tests/100.phpt.

TysonAndre commented:
I added a test of zend_printf to also cover variadics with function pointers and changed the way printf existence was checked for.

There's a difference here - printf is a C function, zend_printf is a C function **pointer** - so I'm not sure if the new test is covering the right thing on windows

nikic commented:
Merged as https://github.com/php/php-src/commit/cdfe11f80d5affd6d5386d84dff0d81323b60bc4, thanks.
